README.md

```diff
@@ -138,8 +138,10 @@ The BibTeX is below:
 }
 ```
 
-For the graph transformation/normalization work, please use the
-following:
+For the graph transformation/normalization work, please cite [Goodman, 2019].
+The BibTeX is below:
+
+[Goodman, 2019]: http://jaslli.org/files/proceedings/05_paclic33_postconf.pdf
```

Comment:
It looks like the `https` link works, too. Can you change it?

```suggestion
[Goodman, 2019]: https://jaslli.org/files/proceedings/05_paclic33_postconf.pdf
```

Reply:
Updated. Thank you